snapcraft/tests/unit/commands/test_version.py

```diff
@@ -19,7 +19,10 @@
 
 
 class VersionCommandTestCase(CommandBaseTestCase):
-
     def test_has_version(self):
         result = self.run_command(['--version'])
         self.assertThat(result.exit_code, Equals(0))
+
+    def test_has_version_without_hyphens(self):
```

Comment:
I'd like to see one more test ensuring that the output from `--version` == the output from `version`. Because I can tell you right now, it's not :wink: .